# HG changeset patch
# User Rik <rik@octave.org>
# Date 1372192935 25200
#      Tue Jun 25 13:42:15 2013 -0700
# Node ID c728c87f43ba9a7d0587ce091c4e7cfc0e974fc3
# Parent  77c8aae15908d80ade7add577907c0d8317ccbd8
ribbon.m: Fix typo of extra spaces in docstring.

* scripts/plot/ribbon.m: Fix typo of extra spaces in docstring.

diff --git a/scripts/plot/ribbon.m b/scripts/plot/ribbon.m
--- a/scripts/plot/ribbon.m
+++ b/scripts/plot/ribbon.m
@@ -15,17 +15,17 @@
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn  {Function File} {} ribbon (@var{x}, @var{y}, @var{width})
 ## @deftypefnx {Function File} {} ribbon (@var{y})
 ## @deftypefnx {Function File} {@var{h} =} ribbon (@dots{})
-## Plot a ribbon plot for the columns of @var{y} vs.  @var{x}.  The
+## Plot a ribbon plot for the columns of @var{y} vs. @var{x}.  The
 ## optional parameter @var{width} specifies the width of a single ribbon
 ## (default is 0.75).  If @var{x} is omitted, a vector containing the
 ## row numbers is assumed (1:rows (Y)).
 ##
 ## The optional return value @var{h} is a vector of graphics handles to
 ## the surface objects representing each ribbon.
 ## @end deftypefn
 
